src/compressor/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(compressor_plugin_dir ${CMAKE_INSTALL_PKGLIBDIR}/compressor)
 add_subdirectory(snappy)
 add_subdirectory(zlib)
 add_subdirectory(zstd)
+add_subdirectory(brotli)
```

Comment:
I'd suggest make this compressor plugin an optional one at this moment. unless we have profiling statistics showing that it outperform other plugins under most circumstances.